packages/rrweb/vite.config.js

```diff
@@ -1,4 +1,28 @@
 import config from '../../vite.config.default';
+import { defineConfig } from 'vite';
+import { mergeConfig } from 'vite';
 
 // export default config('src/index.ts', 'rrweb', { outputDir: 'dist/main' });
-export default config('src/index.ts', 'rrweb');
+const baseConfig = config('src/index.ts', 'rrweb');
+
+export default defineConfig((configEnv) =>
+    mergeConfig(
+        baseConfig(configEnv),
+        {
+            plugins: [{
+                name: 'move-worker-sourcemap',
+                generateBundle(options, bundle) {
+                    Object.entries(bundle).forEach(([fileName, output]) => {
+                        if (fileName.includes('worker') && fileName.endsWith('.map')) {
+                            console.log('Moving worker sourcemap:', fileName);
+                            const newFileName = fileName.replace('assets/', '');
```

Comment:
vite insists on having this map file
and insists on putting it in an assets folder 🤷